README.md

```diff
@@ -1 +1,54 @@
-# Test-Todo-list
\ No newline at end of file
+# Todo-List
+
+> In this project we build a Goal Manager application in which users can add goals and interact with them 
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Tailwind.css
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+- Clone repository
+- Open terminal inside Todo-List-js
+
+```
+npm install
+```
+
+### Usage
+
+```
+npm start
+```
+
```

Comment:
- [ ] [MANDATORY] Don't forget to add tests instructions.

Reply:
@Naqhid kindly add the command to run the test as mentioned by the previous reviewer.